yo1k/tic_tac_toe/core/game.py

```diff
@@ -63,8 +64,18 @@ def size() -> int:
     def __empty_cells() -> list[list[Optional[Mark]]]:
         return [[None for _ in range(Board.size())] for _ in range(Board.size())]
 
-    def __str__(self):
-        return (f"{type(self)}("
+    @staticmethod
+    def assert_board(cells: Optional[Sequence[list[Optional[Mark]]]], expected=None):
```

Comment:
1. Making `cells` `Optional` makes sense only if `None` may be passed as an argument. Here it's not the case.
2. Why is this method public?
3. There is no type specified for the `expected` parameter.
4. There is no possible reason to make `expected` optional. `assert_board`(self.cells, Board.size())` is how this method should be called.

Reply:
Done.